pyupgrade.py

```diff
@@ -1197,6 +1207,9 @@ def _replace_attr(i, mapping, node):
                 func_args, end = _parse_call_args(tokens, i + 1)
                 template = SIX_CALLS[node.func.id]
                 _replace_call(tokens, i, end, func_args, template)
+        elif token.offset in visitor.literal_attrs:
+            func_args, end = _parse_call_args(tokens, i + 3)
+            _replace_call(tokens, i, end, func_args, 'b{args[0]}')
```

Comment:
if it's this easy (?) I think this can just be an entry in the `SIX_CALLS` mapping above

Reply:
oh except it has to be a literal 🤔

Reply:
It seems like using a different set of attrs is a cleaner way to implement this special case